modules/ossm-cert-manager-installation.adoc

```diff
@@ -6,7 +6,7 @@
 [id="ossm-cert-manager-installation_{context}"]
 = Installing cert-manager
 
-To install cert-manager, follow these steps:
+You can install `cert-manager` to manage the lifecycle of TLS certificates; ensuring they are valid and up-to-date. If you are running Istio in your environment, you can also install `istio-csr`, a certificate authority (CA) server that handles certificate signing requests (CSR) from Istio proxies. The `istio-csr` CA delegates signing to cert-manager and cert-manager delegates to the configured CA.
```

Comment:
Per https://61528--docspreview.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-security.html#ossm-cert-manager-integration-istio_ossm-security I'd make this consistent and instead go with "cert-manager tool" or "`cert-manager` tool" throughout the docs